OTKit/otkit-grids/token.yml

```diff
@@ -0,0 +1,20 @@
+props:
+  # These grid values are for viewports between 768px and 1023px.
```

Comment:
Are we missing an intermediate set of values?

Reply:
The screenshots above suggest that we are *not* missing them... however I don't understand why we don't just document this as `< 768` and `≥ 768`?

Reply:
Because when I write notes, I want to mirror what design have. While what you said is true, it isn't how design spec view things. Design spec has 3 viewport ranges, so we document them as is, not our interpretation of it.

Reply:
@jrolfs I @mentioned you in Abstract where these assets were pulled from. It should bring some clarity to why what is what.

Reply:
@mannionaco would you mind approving this PR if you feel like you've addressed the concerns about the discussion above?